src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/model/libra/LibraHearing.java

```diff
@@ -56,6 +57,9 @@ public class LibraHearing {
     private final String courtRoom;
     private final LocalDateTime sessionStartTime;
 
+    @JsonProperty(value="cId")
+    private final String cId;
+
```

Comment:
Are you sure about this?
https://github.com/ministryofjustice/crime-portal-gateway/blob/707199469882e2687971ac06d76dcdd263fdac9f/src/main/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/model/externaldocumentrequest/Case.kt#L22